flight/Modules/Stabilization/stabilization.c

```diff
@@ -84,16 +84,20 @@
 #define HORIZON_MODE_MAX_BLEND               0.85f
 
 enum {
-	PID_RATE_ROLL,   // Rate controller settings
+	PID_RATE,   // Rate controller settings
```

Comment:
It took me a second to see where you were going with this, but only because the name ```PID_RATE``` is pretty broad. I don't know if there's a convention for naming this grouping something like ```_BASE```, ```_GROUP```, ```_BLOCK```, etc...